app/main.py

```diff
@@ -1,2 +1,121 @@
+from typing import Any
+
+
+class HashTable:
+    def __init__(self, key: Any, value: any) -> None:
+        self.key = key
+        self.value = value
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.table = [None] * self.capacity
+        self.load_factor = 0.75
```

Comment:
if we follow dictionary original implementation its load_factor is 2/3 - can be more accurately represented by `Fraction(2, 3)` this class can be used with mathematical operations from the box